api/turing/testdata/cluster/servicebuilder/router_version_missing_bigquery.json

```diff
@@ -54,21 +54,24 @@
     },
     "ensembler": {
         "id": 300,
-        "image": "asia.gcr.io/gcp-project-id/echo:1.0.2",
-        "resource_request": {
-            "min_replica": 1,
-            "max_replica": 2,
-            "cpu_request": "200m",
-            "memory_request": "256Mi"
-        },
-        "endpoint": "echo?delay=10ms",
-        "timeout": "2s",
-        "port": 8080,
-        "env": [
-            {
-                "name": "TEST_ENV",
-                "value": "ensembler"
-            }
-        ]
```

Comment:
Updated the schema to reflect the current schema of an ensembler. The previous schema was incorrect but it didn't break anything because another (earlier) section of the router config was causing the test to fail as expected.